rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1233,11 +1241,11 @@ public String getJobHtml(@HeaderParam("sessionid") String sessionId, @PathParam(
             @PathParam("jobid") String jobId, @PathParam("tasktag") String taskTag)
             throws NotConnectedRestException, UnknownJobRestException, PermissionRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)